setup.py

```diff
@@ -47,25 +29,11 @@ def run(self):
     for item in os.listdir(os.path.dirname(os.path.abspath(__file__)))
     if (os.path.isfile(item) and item.startswith("Dockerfile-"))
 ]
-setup_files = ["setup.py", ".dockerignore", "pyproject.toml", "MANIFEST.in"]
-all_extra_files = dockerfiles + setup_files
-all_extra_files_pathcorrected = [os.path.join("../", item) for item in all_extra_files]
-# find_packages should only ever find these as subpackages of gandlf, not as top-level packages
-# generate this dynamically?
-# GANDLF.GANDLF is needed to prevent recursion madness in deployments
+
+# Any extra files should be located at `GANDLF` module folder (not in repo root)
+extra_files = []
 toplevel_package_excludes = [
-    "GANDLF.GANDLF",
-    "anonymize",
-    "cli",
-    "compute",
-    "data",
-    "grad_clipping",
-    "losses",
-    "metrics",
-    "models",
-    "optimizers",
-    "schedulers",
-    "utils",
+    "testing*",
 ]
```

Comment:
They are not a top-level packages anymore (as they are located **inside** top-level GANDLF package). However, the `testing` is a top-level (together with `GANDLF`); thus we need to exclude it from a wheel (right now a wheel installs a `testing` module also)